src/Microsoft.OpenApi.OData.Reader/Operation/ComplexPropertyDeleteOperationHandler.cs

```diff
@@ -24,9 +24,6 @@ protected override void SetBasicInfo(OpenApiOperation operation)
         // Summary
         operation.Summary = $"Delete {ComplexPropertySegment.Property.Name} property value";
 
-        // Description
-        operation.Description = Context.Model.GetDescriptionAnnotation(ComplexPropertySegment.Property);
```

Comment:
I'm confused to why we're removing the description for those operations?

Reply:
If you check the [diff](https://github.com/microsoft/OpenAPI.NET.OData/pull/178/files#diff-dcf875e9d85634aeb21f747c8f18c1aa7d1372eb3ee45e275eedfe0a6325df53) for the _TripService.OpenApi_ file and inspect the original operations descriptions for the `Delete`, `Update` and `Post` operations, you will notice that the text is somewhat misleading with regard to the operation's intention and the operation summary text. These generic default descriptions only seem to make sense for `Get` operations.
This is my subjective viewpoint.

Reply:
Understood, thanks for the additional context. If the library is not reading the right information today, shouldn't it try to read better information from somewhere else instead of not reading anything at all? What I'm worried about is that after shipping this we get an issue to have descriptions for those operations. Alternatively we could still place a generic description, but a bit more explicit.

Reply:
For the descriptions of operations of complex properties, I propose we use the descriptions from the CRUD restriction annotations targeting complex properties. We are already going to target complex properties with CRUD restrictions annotations in the fix that addresses this issue [only add complextype property path items when CRUD restrictions are present](https://github.com/microsoft/OpenAPI.NET.OData/issues/176) anyway.

But however, for non-GET operations, I propose we still don't use the default fallback description: `Context.Model.GetDescriptionAnnotation(ComplexPropertySegment.Property)`. This way, we provide an avenue for adding CRUD-specific descriptions for complex properties and avoid outputting  default descriptions that are somewhat misleading with regard to the intent of the particular operation.

Reply:
couldn't we have both? get the description from the restriction annotation that matches the operation when present, and if we don't find anything, revert back to the property description with some formatting?

Reply:
What sort of formatting?

Reply:
`Gets the <PropertyName> value for the <ParentTypeName>.` or something along those lines?

Reply:
That feels similar to what we have in the operation `summary` texts.
@darrelmiller do you have any opinions with regard to this?

Reply:
The summary text is fairly clear.  I agree that putting the model property description as the operation description is confusing.  Once we have the restriction annotations on complex types, then we could pull the description from there.  Having both the summary and description mechanically generated doesn't seem to add value in my opinion.